src/client/components/MessageContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Message} from 'fluxxchat-protokolla';
+import '../styles.css';
+
+interface Props {
+	message: Message;
+	clientName: string;
+}
+
+class MessageContainer extends React.Component<Props> {
+
+	public render() {
+		let message;
+		let msg = this.props.message;
```

Comment:
Use `const` instead of `let` if you're not modifying the variable later. `const` tells me immediately that it will not change later.